packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/index.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-ui-angular';
+
+import component from './monitoring.component';
+
+const moduleName = 'ovhManagerDedicatedServerMonitoring';
+
+angular
+  .module(moduleName, ['oui', 'pascalprecht.translate', 'ui.router'])
+  .component('dedicatedServerMonitoring', component)
+  .run(/* @ngTranslationsInject:json ../translations */);
```

Comment:
```suggestion
  .run(/* @ngTranslationsInject:json ./translations */);
```
Could translations be brought by the module itself ?

Reply:
It would require way too much work unfortunately 😞